### PR TITLE
New Elm version

### PR DESCRIPTION
**Pull request by lucamug**

### CODE REVIEW

File: cmd/notes-from-governance-rex.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Rakuten Logo + "Open Source" is ダメ
```
Comment (NemoOudeis) [inline cmd/notes-from-governance-rex.txt]: i think these are internal notes and don't need to be here, do they?

- [ ] remove internal notes

File: docs/test-api/egison.json [hunk 2]
```diff
@@ -0,0 +1,1226 @@
+[
```
Comment (NemoOudeis) [inline docs/test-api/egison.json]: do we need all these json files in the `test-api` folder`?
those look like dumps from the API, but we should hit the API on page load, right?

File: src/Internal/Icon.elm [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+            )
+
+        Star ->
+            ( "0 0 14 16"
+            , [ Svg.path [ saFill cl, SA.fillRule "evenodd", SA.d "M14 6l-4.9-.64L7 1 4.9 5.36 0 6l3.6 3.26L2.67 14 7 11.67 11.33 14l-.93-4.74L14 6z" ] [] ]
+            )
+
+
+
+{- Logo_R ->
```
Comment (NemoOudeis) [inline src/Internal/Icon.elm]: do we need to keep this commented code?

File: src/Internal/Init.elm [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      , indexForRepo = Search.indexBuilderforRepo Repos.repos
+      , repos = Repos.repos
+      }
+    , Cmd.batch <|
+        List.map
+            (\{ githubId, label } ->
+                let
+                    apiUrl =
+                        if True then
+                            -- if url.host == "localhost" then
```
Comment (NemoOudeis) [inline src/Internal/Init.elm]: is this commented out intentionally?

File: docs/test-api/egison.json (hunk 2 above)
Comment (lucamug) [reply docs/test-api/egison.json]: These are used during development so I would keep them there

File: src/Internal/Init.elm (hunk 4 above)
Comment (lucamug) [reply src/Internal/Init.elm]: this was a temporary change for testing, I will restore the original line that was commented out

File: src/Internal/Icon.elm (hunk 3 above)
Comment (lucamug) [reply src/Internal/Icon.elm]: No, I will remove it

File: docs/test-api/egison.json (hunk 2 above)
Comment (NemoOudeis) [reply docs/test-api/egison.json]: hmmm fair enough.
To me it feels a bit iffy to put this test data into the production output.
I think it's fine to have it in the source but after generating the production output I would expect it to be not there. But it's not super important.

**Pull request merged**